e2e/client/playwright/page-object-models/authoring.ts

```diff
@@ -21,4 +21,13 @@ export class Authoring {
             .getByRole('button', {name: actionPath[actionPath.length - 1]})
             .click();
     }
+
+    async openMediaMetadataEditor(field: string = 'media') {
```

Comment:
Could you remove the default? It's a looks messy and might imply there's some default behavior too in the code being tested when there is none. Also it'd be better to have named arguments so one doesn't need to go to definition to see what the string means.

Reply:
it's actually not media field withing text item but just media preview in picture item, so actually there will be only 1 and it has no specific name

Reply:
I see. Isn't it the same component as it would be in a text item having media? If it is, then we should be thinking from a perspective that this is a authoring page object model -> authoring view has a content profile -> content profiles can have multiple media fields. Image content profile can only have one media field, but since this is page object model for authoring, not for image profile, we can't make an assumption here.

Reply:
we have different component for associated media and for media within picture/audio/video item, they have some actions in common but in text item you can drag and drop there other picture items, you can remove the associated picture, that's not available here

Reply:
I see. If they are different components, the question then is how to communicate this using naming/structure/comments of page object models so it's obvious by reading a function signature.

I think having a separate page object model for authoring view with non-text content profiles would be best. Then we wouldn't allow arguments to `openMediaMetadataEditor` and similar functions and it would be clear that only one item is allowed.